pylint/testutils/functional/lint_module_output_update.py

```diff
@@ -0,0 +1,42 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+import csv
+import os
+from typing import Optional
+
+from _pytest.config import Config
+
+from pylint.constants import PY38_PLUS
+from pylint.testutils.functional.test_file import FunctionalTestFile
+from pylint.testutils.lint_module_test import LintModuleTest
+
+
+class LintModuleOutputUpdate(LintModuleTest):
+    """If message files should be updated instead of checked."""
+
+    class TestDialect(csv.excel):
+        delimiter = ":"
+        lineterminator = "\n"
+
+    csv.register_dialect("test", TestDialect)
+
+    def __init__(
+        self, test_file: FunctionalTestFile, config: Optional[Config] = None
+    ) -> None:
+        if not PY38_PLUS:
+            raise RuntimeError(
+                "We need at least python 3.8 for the functional test update to work"
+                ". There is a new, better AST in python 3.8 we're using it exclusively."
```

Comment:
```suggestion
                "You need at least python 3.8 for the functional test updater to work. "
                "This is because python 3.8 includes a new AST parser, which amongst others "
                "returns the end line and end column of most nodes."
```